revolver-config-example.yaml

```diff
@@ -13,6 +13,11 @@ defaults:
       csv:
         file: "audit.csv"
         append: true
+    excludeResources:
+      - tag:
+          name: "Schedule"
+          contains: "ignore"
+      - region: "us-east-1"
```

Comment:
Maybe add a comment saying "exclude resources with Tag:Schedule=ignore AND in us-east-1"
(assuming it's AND not OR)

Reply:
Readme for this references the Filter section which describes what happens; If an array, AND.